packages/wdio-cli/src/constants.js

```diff
@@ -99,6 +99,7 @@ export const QUESTIONNAIRE = [{
     message: 'Where is your automation backend located?',
     choices: [
         'On my local machine',
+        'In the Experitest cloud',
```

Comment:
How about we align the wording to 
```suggestion
        'In the cloud using Experitest',
```